Configuration/StandardSequences/python/DigiToRaw_cff.py

```diff
@@ -47,7 +47,8 @@
 run2_GEM_2017.toReplaceWith(DigiToRawTask, _gem_DigiToRawTask)
 
 from Configuration.Eras.Modifier_run3_GEM_cff import run3_GEM
-run3_GEM.toReplaceWith(DigiToRawTask, _gem_DigiToRawTask)
+from Configuration.Eras.Modifier_phase2_GEM_cff import phase2_GEM
+(run3_GEM & ~phase2_GEM).toReplaceWith(DigiToRawTask, _gem_DigiToRawTask)
```

Comment:
Not sure I understand the need to update this line. Will we have the situation that we will define Run-3 and Phase-2 in the same config? I think the current one in master applied only to Run-3.

Reply:
Phase2 [2] takes run3 [1] and adds phase2_GEM uses phase2_GEM and run3_GEM. For GEM reconstruction, the way its currently working is that run3_GEM turns everything on, and phase2_GEM applies some minor modifications when needed. This has been my understanding anyway.

[1] https://github.com/cms-sw/cmssw/blob/master/Configuration/Eras/python/Era_Run3_cff.py
[2] https://github.com/cms-sw/cmssw/blob/master/Configuration/Eras/python/Era_Phase2_cff.py